include/fileutils.h

```diff
@@ -72,6 +72,25 @@ static inline struct dirent *xreaddir(DIR *dp)
 	return d;
 }
 
+#ifndef HAVE_CLOSE_RANGE
+# ifdef __linux__
+#  include <sys/syscall.h>
+#  if !defined(SYS_close_range) && defined(__NR_close_range)
+   /* Usable kernel-headers, but old glibc-headers.
+    * Call number is expected to be 436. */
+#   define SYS_close_range __NR_close_range
```

Comment:
You do not need __NR_ if you use UL_CHECK_SYSCALL(). See how we use for example SYS_pidfd_send_signal.